dev/src/main/java/org/eclipse/codewind/intellij/ui/module/CustomOptionsStep.java

```diff
@@ -0,0 +1,145 @@
+/*******************************************************************************
+ * Copyright (c) 2020 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v20.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+
+package org.eclipse.codewind.intellij.ui.module;
+
+import com.intellij.ide.util.projectWizard.ModuleWizardStep;
+import com.intellij.openapi.options.ConfigurationException;
+import com.intellij.openapi.progress.ProgressManager;
+import org.eclipse.codewind.intellij.core.CodewindManager;
+import org.eclipse.codewind.intellij.core.CoreUtil;
+import org.eclipse.codewind.intellij.core.Logger;
+import org.eclipse.codewind.intellij.core.cli.InstallStatus;
+import org.eclipse.codewind.intellij.core.cli.InstallUtil;
+import org.eclipse.codewind.intellij.ui.tasks.InstallCodewindTask;
+import org.eclipse.codewind.intellij.ui.tasks.StartCodewindTask;
+import org.eclipse.codewind.intellij.ui.tasks.UpgradeCodewindTask;
+
+import javax.swing.*;
+import java.awt.*;
+import java.awt.event.ActionListener;
+
+import static org.eclipse.codewind.intellij.ui.messages.CodewindUIBundle.message;
+
+public class CustomOptionsStep extends ModuleWizardStep {
+
+    private final JPanel panel;
+    private final JLabel label;
+    private final JButton button;
+
+    private boolean isStarted;
+
+    public CustomOptionsStep() {
+        this.panel = new JPanel();
+        panel.setLayout(new BoxLayout(panel, BoxLayout.PAGE_AXIS));
+
+        label = new JLabel();
+        label.setAlignmentX(Component.LEFT_ALIGNMENT);
+        panel.add(label);
+
+        button = new JButton();
+        panel.add(button);
+
+        configurePage();
+    }
+
+    private void removeActionListeners(JButton button) {
+        for (ActionListener listener : button.getActionListeners()) {
+            button.removeActionListener(listener);
+        }
+    }
+
+    private void configurePage() {
+        try {
+            CodewindManager.getManager().refreshInstallStatus();
```

Comment:
For performance reasons, we should keep the return value of CodewindManager.getManager(), and use it in the several instances of its use in this method.

Reply:
Changes made